docs/develop/natmod.rst

```diff
@@ -21,7 +21,8 @@ language which can be compiled to stand-alone machine code can be put into a
 
 A native .mpy module is built using the ``mpy_ld.py`` tool, which is found in the
 ``tools/`` directory of the project.  This tool takes a set of object files
-(.o files) and links them together to create a native .mpy files.
+(.o files) and links them together to create a native .mpy files.  It requires
+CPython 3 and the library pyelftools v0.26.
```

Comment:
How important is the version 0.26? Also below the instructions are to just run pip install on it, which might not give that exact version.

Reply:
I tested and it needs at least v0.25.  Now updated here and below.

Reply:
Ok!